roles/testnode/handlers/main.yml

```diff
@@ -19,7 +19,8 @@
   service:
     name: "{{ nfs_service }}"
     state: started 
-    enabled: yes
+# FIXME: vivid fails when trying to enable nfs-server.
+# enabled: yes
```

Comment:
Here's what I got from ansible after a fresh re-image to vivid.

```
msg: Error when trying to enable nfs-kernel-server: rc=1 Synchronizing state for nfs-kernel-server.service with sysvinit using update-rc.d...
Executing /usr/sbin/update-rc.d nfs-kernel-server defaults
Executing /usr/sbin/update-rc.d nfs-kernel-server enable
Failed to execute operation: No such file or directory
```

Here's the output from ``sudo service nfs-server status`` before reimage.

```
● nfs-server.service - NFS server and services
   Loaded: loaded (/lib/systemd/system/nfs-server.service; enabled; vendor preset: enabled)
   Active: inactive (dead) since Wed 2015-06-17 12:30:18 EDT; 6s ago
  Process: 28175 ExecStopPost=/usr/sbin/exportfs -f (code=exited, status=0/SUCCESS)
  Process: 28172 ExecStopPost=/usr/sbin/exportfs -au (code=exited, status=0/SUCCESS)
  Process: 28169 ExecStop=/usr/sbin/rpc.nfsd 0 (code=exited, status=0/SUCCESS)
 Main PID: 23270 (code=exited, status=0/SUCCESS)

Jun 04 12:04:25 magna005 systemd[1]: Starting NFS server and services...
Jun 04 12:04:25 magna005 exportfs[23267]: exportfs: can't open /etc/exports for reading
Jun 04 12:04:25 magna005 systemd[1]: Started NFS server and services.
Jun 04 12:22:01 magna005 systemd[1]: Started NFS server and services.
Jun 17 12:30:18 magna005 systemd[1]: Stopping NFS server and services...
Jun 17 12:30:18 magna005 systemd[1]: Stopped NFS server and services.
```

Here's that same output after re-image:

```
● nfs-server.service - NFS server and services
   Loaded: loaded (/lib/systemd/system/nfs-server.service; enabled; vendor preset: enabled)
   Active: active (exited) since Wed 2015-06-17 13:35:06 EDT; 11min ago
 Main PID: 22132 (code=exited, status=0/SUCCESS)
   CGroup: /system.slice/nfs-server.service

Jun 17 13:35:06 magna005 systemd[1]: Starting NFS server and services...
Jun 17 13:35:06 magna005 exportfs[22129]: exportfs: can't open /etc/exports for reading
Jun 17 13:35:06 magna005 systemd[1]: Started NFS server and services.
```